src/App.js

```diff
@@ -1,19 +1,9 @@
-/**
- * It renders the AppCore component with the correct theme.
- * @returns The App component is being returned.
- */
-import AppThemeSelector from './components/Themes/AppThemeSelector'
+import './stylesheets/font-awesome.min.css'
+import './stylesheets/AppTheme.css'
 import AppCore from './components/Core/AppCore'
 
 function App() {
-  const hash = document.location.hash.replace('#', '')
-  const agentIndex = hash === '' ? '1' : hash
-
-  return (
-    <AppThemeSelector agent={agentIndex}>
-      <AppCore agent={agentIndex} />
-    </AppThemeSelector>
-  )
+  return <AppCore />
```

Comment:
glad we are no longer passing agents, found it very confusing

Reply:
yep really confusing